client/src/main/java/io/pravega/client/segment/impl/SegmentInputStreamImpl.java

```diff
@@ -178,7 +178,9 @@ private void issueRequestIfNeeded() {
         //compute read length based on current offset up to which the events are read.
         int updatedReadLength = computeReadLength(offset + buffer.dataAvailable());
         if (!receivedEndOfSegment && !receivedTruncated && updatedReadLength > 0 && outstandingRequest == null) {
-            log.trace("Issuing read request for segment {} of {} bytes", getSegmentId(), updatedReadLength);
+            if (log.isTraceEnabled()) {
```

Comment:
Is this faster compared to what it was?

Reply:
It showed up in the flamegraph before, so yes.